stacker/util.py

```diff
@@ -459,19 +504,22 @@ class SourceProcessor():
     """Makes remote python package sources available in the running python
        environment."""
 
-    def __init__(self, stacker_cache_dir=None):
+    def __init__(self, sources, stacker_cache_dir=None):
         """
-        Processes a config's list of package sources
+        Processes a config's defined package sources.
 
         Args:
-            stacker_cache_dir (string): Directory of stacker local cache.
-                                        Default to $HOME/.stacker
+            sources (dict): Package sources from Stacker config dictionary
+            stacker_cache_dir (string): Path where remote sources will be
+                cached.
         """
-        stacker_cache_dir = (stacker_cache_dir or
-                             os.path.expanduser("~/.stacker"))
+        if stacker_cache_dir is None:
```

Comment:
probably should be: `if not stacker_cache_dir:` - or the previous method of using `or` is fine too.